### PR TITLE
[SQL Server] (No.8) add sql server UI java codes related to show propertes.

### PR DESCRIPTION
**Pull request by shenqianjin**

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/sqlserver/common/SqlServerDatabaseComboBox.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class SqlServerDatabaseComboBox extends AzureComboBox<SqlDatabaseEntity> {
+
+    private ISqlServer server;
+
+    public void setSubscription(ISqlServer server) {
```
Comment (andxu) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/sqlserver/common/SqlServerDatabaseComboBox.java:21]: ???

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/sqlserver/common/SqlServerDatabaseComboBox.java [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+import com.microsoft.azure.toolkit.intellij.common.AzureComboBox;
+import com.microsoft.azure.toolkit.lib.common.operation.AzureOperation;
+import com.microsoft.azure.toolkit.lib.sqlserver.model.SqlDatabaseEntity;
+import com.microsoft.azure.toolkit.lib.sqlserver.service.ISqlServer;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class SqlServerDatabaseComboBox extends AzureComboBox<SqlDatabaseEntity> {
```
Comment (andxu) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/sqlserver/common/SqlServerDatabaseComboBox.java:17]: I don't see any valuable thing in this class than DatabaseComboBox

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/sqlserver/properties/SqlServerPropertyView.java [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+import org.jetbrains.annotations.NotNull;
+
+import javax.swing.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ItemEvent;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class SqlServerPropertyView extends BaseEditor implements MvpView {
```
Comment (andxu) [inline PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/sqlserver/properties/SqlServerPropertyView.java:46]: add todo: shared common code

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/sqlserver/common/SqlServerDatabaseComboBox.java (hunk 1 above)
Comment (shenqianjin) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/sqlserver/common/SqlServerDatabaseComboBox.java:21]: removed.

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/sqlserver/properties/SqlServerPropertyView.java (hunk 3 above)
Comment (shenqianjin) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/sqlserver/properties/SqlServerPropertyView.java:49]: added

**Review by andxu — APPROVED** [review]

**Comment by andxu** [comment]
Please refactor the code with MySQLPropertryOverviewPanel

File: PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/sqlserver/common/SqlServerDatabaseComboBox.java (hunk 2 above)
Comment (shenqianjin) [reply PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/sqlserver/common/SqlServerDatabaseComboBox.java:20]: added todo to track for refacor.

**Pull request merged**